code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -419,6 +419,15 @@
 	heat_protection = CHEST|GROIN|ARMS|LEGS
 	cold_protection = CHEST|GROIN|ARMS|LEGS
 
+/obj/item/clothing/suit/jacket/leather/duster/armoured
```

Comment:
this needs a different name or icon so you can tell whether its armored or not

Reply:
changed it to "Armoured leather duster"